### PR TITLE
yungong - try to add CommonsPlay exception test

### LINKED ISSUE #58: Adding exception test for CommonsPlay
There is a stryker disable comment here, and I will remove it and add a test for the field of `[welcomeText, _]`.
https://ucsb-cs156-f24.slack.com/archives/C080R66E326/p1732552002315739

before:
image
after:
image

I really also want to add test for `firstname`, however it involves some twisted optional chaining, which I think is impossible to test, so I just added test for the second line.
image

### PR DESCRIPTION
**Pull request by yungongWang**
## Overview

I add another test for `CommonsPlay.js`. There is a stryker disable comment here, and I removed it and add a test for the field of `[welcomeText, _]`.
https://ucsb-cs156-f24.slack.com/archives/C080R66E326/p1732552002315739

## Screenshots (Optional)

before:
image
after:
image

## Future Possibilities (Optional)

I really also want to add test for `firstname`. However it involves some twisted optional chaining, which I think is impossible to test (at least incapable of me), so I just added test for the `[welcomeText, _]`. I added two tests in total, and I believed one of them can resolve the one of four mutation tests for `firstname` generated by GitHub auto-test, although I still disabled it. But, this test can help if somebody also want to add test for `firstname` in the future, so I kept it. 
image
This test can kill one of the mutation tests for `firstname`.
image

## Validation (Optional)

1. All the tests are passed.

## Tests

- [x] Backend Unit tests (`mvn test`) pass
- [x] Backend Test coverage (`mvn test jacoco:report`) 100%
- [x] Backend Mutation tests (`mvn test pitest:mutationCoverage`) 100% 
- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues

Closes #58

### CODE REVIEW

File: frontend/src/tests/components/Commons/CommonsPlay.test.js [hunk 1]
```diff
@@ -35,4 +36,29 @@ describe("CommonsPlay tests", () => {
             expect(screen.getByTestId("commons-card")).toBeInTheDocument();
         });
     });
+    test("Tests for not logged in", async () => {
```
Comment (yungongWang) [inline frontend/src/tests/components/Commons/CommonsPlay.test.js:39]: This test can kill one of the mutation tests for `firstname`.

**Review by jzipkin1812 — APPROVED** [review]
Seems like you correctly took a line of code out of an existing Stryker comment and added a test for it. Everything is passing

File: frontend/src/tests/components/Commons/CommonsPlay.test.js (hunk 1 above)
Comment (pconrad) [reply frontend/src/tests/components/Commons/CommonsPlay.test.js:39]: Nice

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**